### PR TITLE
Xw 923 | Global Nav 2016

### PR DESCRIPTION
**Pull request by bognix**
@Wikia/x-wing 

Related PR: https://github.com/Wikia/config/pull/1531

### CODE REVIEW

File: extensions/wikia/WikiaLogo/WikiaLogoHelper.class.php [hunk 1]
```diff
@@ -51,4 +53,28 @@ public function getCentralWikiUrlForLangIfExists( $lang ) {
...
+		}
+	}
+
+	public function isFandomExposed() {
+		global $wgEnableGlobalNav2016, $wgLanguageCode;
+
+		if ( empty( $wgEnableGlobalNav2016 ) ) {
+			return false;
+		} else {
+			if ( $wgLanguageCode === self::FANDOM_LANG ) {
```
Comment (hakubo) [inline extensions/wikia/WikiaLogo/WikiaLogoHelper.class.php]: return $wgLanguageCode === self::FANDOM_LANG; ?

File: extensions/wikia/GlobalNavigation/GlobalNavigationController.class.php [hunk 2]
```diff
@@ -24,28 +24,33 @@ public function __construct() {
...
+		$this->response->setVal( 'notificationsEnabled', !empty( $userCanRead ) );
+		$this->response->setVal( 'isAnon', $wgUser->isAnon() );
 
 		$isGameStarLogoEnabled = $this->isGameStarLogoEnabled();
 		$this->response->setVal( 'isGameStarLogoEnabled', $isGameStarLogoEnabled );
+		$this->response->setVal( 'isFandomExposed', $this->wikiaLogoHelper->isFandomExposed() );
 		if ( $isGameStarLogoEnabled ) {
 			$this->response->addAsset( 'extensions/wikia/GlobalNavigation/styles/GlobalNavigationGameStar.scss' );
 		}
+		$this->response->setVal( 'menuContents', $this->helper->getMenuNodes2016() );
```
Comment (tomasznapieralski) [inline extensions/wikia/GlobalNavigation/GlobalNavigationController.class.php]: do you want to run this also for nav 2015?

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+		'jquery', 'wikia.window', 'wikia.delayedhover', 'wikia.globalNavigationDropdowns'
+	],
+	function ($, w, delayedHover, dropdowns) {
+		'use strict';
+
+		var $entryPoint,
+			$exploreLink;
+
+		/**
+		 * @desc Handle click on entry point for logged in users.
```
Comment (tomasznapieralski) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js]: is `@desc` required? or it is old habbit?

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+		 * Second click on entry point for logged in users is redirecting to user profile page.
+		 * This method should be removed after we unify the ux for anon and logged in.
+		 * @param {Event} event
+		 */
+		function onEntryPointClick(event) {
+			var $this = $(event.target);
+			event.preventDefault();
```
Comment (tomasznapieralski) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js]: please provide line above

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+			event.stopImmediatePropagation();
+			if (this.$entryPoint.hasClass('active')) {
```
Comment (tomasznapieralski) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js]: please provide line above

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+				w.location = $this.attr('href');
+			} else {
+				dropdowns.openDropdown.call(this.$entryPoint.get(0));
+			}
+		}
+
+		$(function () {
+			$entryPoint = $('#exploreWikiaEntryPoint');
```
Comment (tomasznapieralski) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js]: please use consistent way to access this function variables.
use `this.$entryPoint` or `$entryPoint` in whole file

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js (hunk 5 above)
Comment (tomasznapieralski) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js]: please use consistent way to access this function variables.
use `this.$entryPoint` or `$entryPoint` in whole file

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+		 * @param {Event} event
+		 */
+		function onEntryPointClick(event) {
+			var $this = $(event.target);
+			event.preventDefault();
+			event.stopImmediatePropagation();
+			if (this.$entryPoint.hasClass('active')) {
+				w.location = $this.attr('href');
+			} else {
+				dropdowns.openDropdown.call(this.$entryPoint.get(0));
```
Comment (tomasznapieralski) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js]: please use consistent way to access this function variables.
use `this.$entryPoint` or `$entryPoint` in whole file

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+			}
+		}
+
+		$(function () {
+			$entryPoint = $('#exploreWikiaEntryPoint');
+			$exploreLink = $entryPoint.children('a.cell-link');
```
Comment (tomasznapieralski) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js]: please use consistent way to access this function variables.
use `this.$exploreLink` or `$exploreLink` in whole file

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js (hunk 3 above)
Comment (vforge) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js]: According to guidelines, shouldn't be here (I assume old habit)

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss [hunk 9]
```diff
@@ -91,5 +92,142 @@
...
+	.notifications-container .notifications-container,
+	.account-navigation-container .user-menu,
+	.AccountNavigation .UserLoginDropdown {
+		margin-top: 1px;
+	}
+
+	.wikia-logo {
+		display: flex;
+		align-items: flex-end;
+		text-decoration: none !important;
```
Comment (tomasznapieralski) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss:105]: is this `!important` rly important?

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss [hunk 10]
```diff
@@ -91,5 +92,142 @@
...
+	.global-nav-search-input-wrapper .autocomplete-container,
+	.notifications-container .notifications-container,
+	.account-navigation-container .user-menu,
+	.AccountNavigation .UserLoginDropdown {
+		margin-top: 1px;
+	}
+
+	.wikia-logo {
+		display: flex;
+		align-items: flex-end;
```
Comment (tomasznapieralski) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss:103]: alphabetic order

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss [hunk 11]
```diff
@@ -91,5 +92,142 @@
...
+			border-top: 1px solid #ddd;
+			color: $global-navigation-text-color;
+			display: block;
+			font-size: 14px;
+			padding: 12px;
+			text-decoration: none;
+			width: 100%;
+
+			&:hover {
+				background-color: #e6ebf2;
```
Comment (tomasznapieralski) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss]: is this color: `$global-navigation-link-hover-color`? or is there variable for that?

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss [hunk 12]
```diff
@@ -91,5 +92,142 @@
...
+					color: $global-navigation-link-hover-color;
+				}
+			}
+		}
+	}
+
+	.explore-wikia-entry-point {
+		> a {
+			position: relative;
+			padding-right: 18px;
```
Comment (tomasznapieralski) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss:167]: alphabetical order

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 13]
```diff
@@ -284,7 +284,17 @@ public function getWikiVariables() {
...
+			$wikiVariables['navigation'] = $navigation;
+		} else {
+			$navigationNodes = (new GlobalNavigationHelper())->getMenuNodes2016();
+			$wikiVariables['hubsLinks'] = $navigationNodes['hubs'];
+			$wikiVariables['explore'] = $navigationNodes['explore'];
+			$wikiVariables['localNav'] = $navigation;
+			$wikiVariables['enableGlobalNav2016'] = $wgEnableGlobalNav2016;
+		}
+
+
```
Comment (tomasznapieralski) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: remove extra line

File: extensions/wikia/GlobalNavigation/GlobalNavigationController.class.php (hunk 2 above)
Comment (vforge) [inline extensions/wikia/GlobalNavigation/GlobalNavigationController.class.php]: No.

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss (hunk 9 above)
Comment (bognix) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss:105]: I believe we should leave it. We don't want to get the "HOME OF FANDOM" link underlined by some user/wiki/old styles. 

`!important` only shows how badly we don't want it to happen

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss (hunk 11 above)
Comment (bognix) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss]: there is no variable for this value

**Comment by tomasznapieralski** [comment]
LGTM

File: extensions/wikia/GlobalNavigation/GlobalNavigationController.class.php (hunk 2 above)
Comment (nandy-andy) [inline extensions/wikia/GlobalNavigation/GlobalNavigationController.class.php]: What about 2017 and future years? :trollface:

File: extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php [hunk 14]
```diff
@@ -94,4 +94,24 @@ protected function createCNWUrlFromGlobalTitle() {
...
+			'global-navigation-menu-hubs',
+			[3] // max 3 links
+		);
+		$exploreNodes = ( new NavigationModel( true /* useSharedMemcKey */ ) )->getTree(
+			NavigationModel::TYPE_MESSAGE,
+			'global-navigation-menu-explore',
+			[1, 3] // max 1 entry with 3 links
+		);
+
+		$menuNodes = [
```
Comment (hakubo) [inline extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php]: you don't really need that var do you?

File: extensions/wikia/GlobalNavigation/GlobalNavigationController.class.php (hunk 2 above)
Comment (bognix) [inline extensions/wikia/GlobalNavigation/GlobalNavigationController.class.php]: don't you worry :+1:

File: extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php (hunk 14 above)
Comment (bognix) [inline extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php]: we don't but for me this is more readable this way. Instead of having huge array definition which IMHO is hard to follow when it expands to 8 lines - we have two separated functions with readable output.

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js [hunk 15]
```diff
@@ -0,0 +1,49 @@
...
+
+			dropdowns.attachDropdown($entryPoint, {
+				onClick: onEntryPointClick.bind({$entryPoint: $entryPoint}),
+				onClickTarget: $exploreLink.get(0)
+			});
+
+			delayedHover.attach(
+				$entryPoint.get(0), {
```
Comment (hakubo) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js:58]: can { go to next line? I think it would be more readable.

File: extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php (hunk 14 above)
Comment (vforge) [inline extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php]: ^ this, exactly

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss [hunk 16]
```diff
@@ -91,5 +92,142 @@
 	display: none;
 }
 
+.global-navigation-2016 {
+	.global-nav-search-input-wrapper .autocomplete-container,
+	.notifications-container .notifications-container,
```
Comment (Warkot) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss]: Are you sure you wanted to write the same class twice here?

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss [hunk 17]
```diff
@@ -91,5 +92,142 @@
...
+		margin-top: 1px;
+	}
+
+	.wikia-logo {
+		align-items: flex-end;
+		display: flex;
+		text-decoration: none !important;
+	}
+
+	.wikia-logo__subtitle {
```
Comment (Warkot) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss:108]: Why not `.wikia-logo-subtitle`?

File: extensions/wikia/GlobalNavigation/templates/GlobalNavigation_index.php [hunk 18]
```diff
@@ -1,41 +1,116 @@
...
+					<img src="<?= $wg->BlankImgUrl ?>"
+						height="24"
+						width="91"
+						alt="<?= wfMessage( 'oasis-global-page-header' )->escaped() ?>"
+						title="<?= wfMessage( 'oasis-global-page-header' )->escaped() ?>">
+				</a>
+			</div>
+			<div class="hubs-container table-cell" id="hubsEntryPoint">
+				<?= $app->renderView( 'GlobalNavigation', 'hubsMenu' ) ?>
+	</div>
```
Comment (Warkot) [inline extensions/wikia/GlobalNavigation/templates/GlobalNavigation_index.php]: [You must construct additional indents](http://knowyourmeme.com/memes/you-must-construct-additional-pylons). (starting from this line)

File: extensions/wikia/GlobalNavigation/templates/GlobalNavigation_index.php [hunk 19]
```diff
@@ -1,41 +1,116 @@
...
+								<li>
+									<a class="cell-link" href="<?= $exploreLink['href']; ?>"><?= $exploreLink['textEscaped']; ?></a>
+								</li>
+							<?php endforeach; ?>
+							<li>
+								<a href="<?= htmlspecialchars( $createWikiUrl ) ?>"
+									class="start-wikia"
+									title="<?= wfMessage( 'global-navigation-create-wiki' )->escaped(); ?>"
+									data-id="start-wikia">
+									<span><?= ( wfMessage( 'global-navigation-create-wiki' )->escaped() ); ?></span>
```
Comment (Warkot) [inline extensions/wikia/GlobalNavigation/templates/GlobalNavigation_index.php]: Why the outer parentheses?

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 20]
```diff
@@ -580,7 +589,7 @@ private function getOtherLanguages( Title $title ) {
 		}
 
 		$url = $title->getFullURL();
-		//$url = str_replace( '.rychu.wikia-dev.com', '.wikia.com', $url );
+		// $url = str_replace( '.rychu.wikia-dev.com', '.wikia.com', $url );
```
Comment (Warkot) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php:616]: Isn't this just a development leftover to be removed before merging to dev?

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 21]
```diff
@@ -602,7 +611,7 @@ private function getOtherLanguages( Title $title ) {
 				'articleTitle' => str_replace( '_', ' ', $articleTitle ),
 				'url' => $url,
 			];
-		}, array_keys( $links ), array_values( $links ) );
+		} , array_keys( $links ), array_values( $links ) );
```
Comment (Warkot) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php:638]: ` ` here? :confused:

File: extensions/wikia/WikiaLogo/WikiaLogoHelper.class.php [hunk 22]
```diff
@@ -51,4 +53,24 @@ public function getCentralWikiUrlForLangIfExists( $lang ) {
 			return false;
 		}
 	}
+
+	public function getMainCorpPageURL() {
+		global $wgLang;
+
+		if (empty($this->isFandomExposed())) {
```
Comment (Warkot) [inline extensions/wikia/WikiaLogo/WikiaLogoHelper.class.php]: Spawn more spaces!

File: extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php (hunk 14 above)
Comment (rogatty) [inline extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php]: I think @hakubo meant that we can do

```
return [
    'hubs' => $hubsNodes,
    'explore' => $exploreNodes[0]
];
```

and it will be readable just like now :)

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js [hunk 23]
```diff
@@ -0,0 +1,49 @@
...
+	function ($, w, delayedHover, dropdowns) {
+		'use strict';
+
+		var $entryPoint,
+			$exploreLink;
+
+		/**
+		 * Handle click on entry point for logged in users.
+		 * Second click on entry point for logged in users is redirecting to user profile page.
+		 * This method should be removed after we unify the ux for anon and logged in.
```
Comment (rogatty) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js]: Is this description correct or was it copied from somewhere else?

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js [hunk 24]
```diff
@@ -0,0 +1,49 @@
...
+		 * @param {Event} event
+		 */
+		function onEntryPointClick(event) {
+			var $this = $(event.target);
+
+			event.preventDefault();
+			event.stopImmediatePropagation();
+
+			if ($entryPoint.hasClass('active')) {
```
Comment (rogatty) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js]: Does it work because on touch devices there is no active class on the first click? Would be nice to have a comment here explaining what's going on.

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss [hunk 25]
```diff
@@ -91,5 +92,142 @@
...
+
+	.explore-wikia-dropdown {
+		background-color: $global-wikia-background-color;
+		border: 1px solid rgba(0,0,0,.15);
+		border-top: none;
+		display: none;
+		position: absolute;
+		transition-duration: .5s;
+		transition-property: background-color;
+		z-index: 5000100;
```
Comment (rogatty) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss]: Wow, isn't that a magic number. Shouldn't we use a variable here?

File: extensions/wikia/MercuryApi/MercuryApiController.class.php (hunk 20 above)
Comment (rogatty) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php:616]: Nah, see https://github.com/Wikia/app/pull/9361#discussion_r48497607

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 26]
```diff
@@ -284,7 +284,16 @@ public function getWikiVariables() {
...
 
-		$wikiVariables['navigation'] = $navigation;
+		if ( empty( $wgEnableGlobalNav2016 ) ) {
+			$wikiVariables['navigation'] = $navigation;
+		} else {
+			$navigationNodes = ( new GlobalNavigationHelper() )->getMenuNodes2016();
+			$wikiVariables['hubsLinks'] = $navigationNodes['hubs'];
+			$wikiVariables['explore'] = $navigationNodes['explore'];
+			$wikiVariables['localNav'] = $navigation;
+			$wikiVariables['enableGlobalNav2016'] = $wgEnableGlobalNav2016;
```
Comment (rogatty) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: Can we put all this data in `$wikiVariables['navigation']`? It is navigation data after all. Also would be nice to have all this logic in the `getNavigation` method.

File: extensions/wikia/WikiaLogo/tests/WikiaLogoTest.php [hunk 27]
```diff
@@ -2,6 +2,9 @@
 class WikiaLogoTest extends WikiaBaseTest
 {
 
+	const FANDOM_URL = WikiaLogoHelper::FANDOM_URL;
+	const CORP_PAGE_URL = 'wikia.com';
```
Comment (rogatty) [inline extensions/wikia/WikiaLogo/tests/WikiaLogoTest.php]: It's not used anywhere.

File: extensions/wikia/WikiaLogo/WikiaLogoHelper.class.php (hunk 22 above)
Comment (hakubo) [inline extensions/wikia/WikiaLogo/WikiaLogoHelper.class.php]: spaces

File: extensions/wikia/MercuryApi/MercuryApiController.class.php [hunk 28]
```diff
@@ -284,7 +284,16 @@ public function getWikiVariables() {
 			);
 		}
 
-		$wikiVariables['navigation'] = $navigation;
+		if ( empty( $wgEnableGlobalNav2016 ) ) {
+			$wikiVariables['navigation'] = $navigation;
+		} else {
+			$navigationNodes = ( new GlobalNavigationHelper() )->getMenuNodes2016();
+			$wikiVariables['hubsLinks'] = $navigationNodes['hubs'];
```
Comment (hakubo) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php]: what do you think about creating an object for all of these?

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss [hunk 29]
```diff
@@ -91,5 +92,142 @@
...
+			padding: 12px;
+			text-decoration: none;
+			width: 100%;
+
+			&:hover {
+				background-color: $global-navigation-cell-hover-color;
+				text-decoration: none;
+			}
+
+			&.start-wikia {
```
Comment (hakubo) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss]: does it have to be that deep?

File: extensions/wikia/MercuryApi/MercuryApiController.class.php (hunk 20 above)
Comment (Warkot) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php:616]: :see_no_evil: I still don't get it, why it's left here. Is it read by some machine/bot/whatever? If not, there's not much difference between uncommenting this thing and just pasting it on dev-/sandbox.

File: extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php (hunk 14 above)
Comment (bognix) [inline extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php]: ah okey - I blame github on phone

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss (hunk 29 above)
Comment (vforge) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss]: yes - it needs to override previous styling. but we could clean this up while removing old (non-2016) nav later

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss (hunk 16 above)
Comment (bognix) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss]: unfortunately yes - this is old notifications code and we are a little afraid to mangle this code.

Comment (vforge) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss]: It's not the same class - it has different parent :) - it's just named same way as other one, but see below - we have 2 more with different names!

File: extensions/wikia/MercuryApi/MercuryApiController.class.php (hunk 21 above)
Comment (bognix) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php:638]: @Warkot script from guidelines put it there - do you still think we should fix it?

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss (hunk 17 above)
Comment (vforge) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss:108]: a BEM-like personal preference - do you think we should change it?

File: extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js (hunk 24 above)
Comment (bognix) [inline extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js]: I've added this info to descirption

File: extensions/wikia/GlobalNavigation/templates/GlobalNavigation_index.php (hunk 18 above)
Comment (bognix) [inline extensions/wikia/GlobalNavigation/templates/GlobalNavigation_index.php]: should be fine now

File: extensions/wikia/MercuryApi/MercuryApiController.class.php (hunk 21 above)
Comment (Warkot) [inline extensions/wikia/MercuryApi/MercuryApiController.class.php:638]: We should then probably fix the script… but maybe another time. ;)

File: extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss (hunk 17 above)
Comment (Warkot) [inline extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss:108]: I supposed so. However, we're not using BEM and our guidelines encourage dashed-case for (s)css.

File: extensions/wikia/GlobalNavigation/templates/GlobalNavigation_index.php (hunk 18 above)
Comment (Warkot) [inline extensions/wikia/GlobalNavigation/templates/GlobalNavigation_index.php]: I don't see any change. :see_no_evil: :confused:

**Comment by bognix** [comment]
unit tests failure is not related to the change - happens on other branches too

**Comment by hakubo** [comment]
LGTM
